bower.json

```diff
@@ -16,8 +16,8 @@
     "package.json"
   ],
   "dependencies": {
-    "purescript-effect": "master",
-    "purescript-integers": "master",
+    "purescript-effect": "^4.0.0",
+    "purescript-integers": "^6.0.0",
     "purescript-math": "master"
```

Comment:
Looks like this got missed, probably because it should be removed altogether.

Reply:
Good point! My script doesn't account for that.